remove-node.yml

```diff
@@ -5,35 +5,18 @@
 - name: Ensure compatibility with old groups
   import_playbook: legacy_groups.yml
 
-- hosts: "{{ node | default('etcd:k8s_cluster:calico_rr') }}"
-  gather_facts: no
-  environment: "{{ proxy_disable_env }}"
-  vars_prompt:
-    name: "delete_nodes_confirmation"
-    prompt: "Are you sure you want to delete nodes state? Type 'yes' to delete nodes."
```

Comment:
Why delete this?

Reply:
I think this is the only playbook execution that asks for confirmation in all kubespray. Why ask for confirmation only in this scenario? I consider it's not necessary. Also, i think this can make it difficult to automate and integrate in some installation pipelines.

Reply:
This kind of confirmation is normal for deleting some objects in general.
It is better to add another option to skip this confirmation if we want without deleting this block.

Reply:
Hi @oomichi , check out my last commit and let me know what you think about the change. Thanks!

Reply:
Thanks for adding the option, that seems good for me.